DataFormats/CSCDigi/interface/CSCCorrelatedLCTDigi.h

```diff
@@ -185,40 +185,40 @@ class CSCCorrelatedLCTDigi {
   // more information, please check "DN-20-016".
 
   // Run-1, Run-2 and Run-3 trknmb is either 1 or 2.
-  uint16_t trknmb;
+  uint16_t trackNumber_;
```

Comment:
is there any old data where CSCCorrelatedLCTDigi should still be readable correctly?
I do not see any ioread rules to preserve backward compatibility.

Reply:
Hi Slava, yes, I need to add the ioread rules. Thanks.